### ISSUE TITLE
Correct minor error in the english text

### ISSUE THREAD

**Issue by LAfricain** [postiffm/bibledit-desktop#117]
In the list of the books, the book number 0 is called ;`**EMPTY ID=0**`:
```
src/bookdata.cpp:38:  "**EMPTY ID=0**",             "",        "",     "",     "",     "",       "",                 0, btUnknown,  false}, {
src/books.cpp:392:  books_table[0].localname = _("**EMPTY ID=0**");   // no abbreviation for this
```
What the corresponding book? Probably the "A7 | INT | Introduction Matter" of the [usfm3 book identification](https://ubsicap.github.io/usfm/identification/books.html).
And during the checks it displays `gtk-cancel`, instead of `Cancel`:
```
templates/gtkbuilder.progressdialog.xml:36:            <property name="label" translatable="yes">gtk-cancel</property>
```
Idem for gtk-open, should be only "Open":
```
src/assistantbackup.cpp:131:  image1 = gtk_image_new_from_stock (_("gtk-open"), GTK_ICON_SIZE_BUTTON);
```
If you agree, I do this corrections.

**Comment by postiffm** [comment]
I put books_table[0] as empty to catch errors in the code. It is important to my design and should NOT be changed. A7 would be a larger number, like 106. But we don't support USFM 3.0 yet. So that is a bridge that we will cross in the future, but not now.

"gtk-open" is a keyword for the GTK library, so _("gtk-open") is an error. It should just say "gtk-open"

Similar with gtk-cancel. It should not be translatable.

**Comment by LAfricain** [comment]
OK then just unlocalised the tree lines. Or for **EMPTY ID=0** maybe a blank place will be better.

**Comment by postiffm** [comment]
I would be happy to integrate a pull request that fixes just these bugs.

**Comment by LAfricain** [comment]
Ok let me do it.

**Comment by LAfricain** [comment]
Gtk-cancel in templates/gtkbuilder.progressdialog.xml need I to remove the translatable or Change for Cancel? Because it is a xml file?

**Comment by postiffm** [comment]
Now that you say that...I think since it is a label, it should be translatable, and it should say "Cancel" and not gtk-cancel.

Is it true that the cancel button has a label that shows the text gtk-cancel on it? I have never seen this that I remember.

**Comment by LAfricain** [comment]
![gtk-cancel](https://user-images.githubusercontent.com/19690477/61577224-a2141000-aae4-11e9-9fa5-39633366f851.png)

**Comment by LAfricain** [comment]
Corrected by #118
